app/controllers/conferences_controller.rb

```diff
@@ -16,12 +16,11 @@ def show
 
   def create
     team = current_user.student_team
```

Comment:
if a current user is not a student, the team will be empty. 
When the conference was saved, the 'team.present?' will return false and then the user will be redirected to a conferences_path after creating a new conference.
However, this button just appears for users that have an ability to 'update_conference_preferences' -  just orga members and students of that specific team.